public/locales/en/translation.json

```
@@ -622,12 +622,32 @@
             "general": "General",
             "log": "Log",
             "other": "Other",
-            "sync": "Cloud Saves Sync"
+            "sync": "Cloud Saves Sync",
+            "systemInformation": "System Information"
         },
         "open-config-file": "Open Config File",
         "reset-heroic": "Reset Heroic",
         "saves": {
             "warning": "Cloud Saves feature is in Beta, please backup your saves before syncing (in case something goes wrong)"
+        },
+        "systemInformation": {
+            "copyToClipboard": "Copy to clipboard",
+            "CPU": "CPU:",
```

```suggestion
            "cpu": "CPU:",
```

for consistency with the rest (gpu, os)